SPECS/WALinuxAgent/WALinuxAgent.spec

```diff
@@ -74,10 +74,12 @@ python3 setup.py check && python3 setup.py test
 %config %{_sysconfdir}/waagent.conf
 %ghost %{_localstatedir}/log/waagent.log
 %dir %attr(0700, root, root) %{_sharedstatedir}/waagent
-%{_lib}/python3.7/site-packages/*
+%{python3_sitelib}/*
 
 %changelog
 * Mon Jun 21 2021 Thomas Crain <thcrain@microsoft.com> - 2.2.54.2-1
```

Comment:
We shouldn't be changing the history for a spec. The original from `1.0-dev` with this version and release number was authored by Suresh and didn't include the lib macro updates. I'd revert to the original changelog entry and just add another release for `dev` on top of it.